### PR TITLE
Add TWPA to RFSoc4x2

### PR DESCRIPTION
**Pull request by rodolfocarobene**
I should have already done this. It also enables testing of the new twpa optimization routine

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibolab/platform.py [hunk 1]
```diff
@@ -188,11 +189,17 @@ def create_tii_rfsoc4x2(runcard, address=None):
     channels["L2-RO"].ports = [("i0", 0)]  # feedback
     channels["L3-18_qd"].ports = [("o1", 1)]  # drive
 
+    local_oscillators = [
+        LocalOscillator("twpa_a", "192.168.0.32"),
+    ]
+    local_oscillators[0].frequency = 6_200_000_000
+    local_oscillators[0].power = -1
```
[the remark this answers is not in the document]
Comment (rodolfocarobene) [reply src/qibolab/platform.py:196]: Yes tii1q-b1 was already using these values, but they were set just manually

**Comment by stavros11** [comment]
Thanks, I didn't know that you can put negative power in the LO. 

In terms of code it looks okay, just note that state-0 and state-1 probabilities calculated by the tests are wrong in this branch (see the [action output](https://github.com/qiboteam/qibolab/actions/runs/4960203362/jobs/8875371717?pr=411)). The same action yesterday gave much [better results](https://github.com/qiboteam/qibolab/actions/runs/4949505803/jobs/8851767053?pr=409).
I am not sure if this is due to the TWPA settings in this PR or something else in the lab, but it's worth investigating before merging.

**Review by stavros11 — APPROVED** [review]
I think the probabilities issue is not related to the TWPA, but maybe you need to re-calibrate the classifcation parameters because the server was restarted.

**Comment by rodolfocarobene** [comment]
Yes, I rebooted the server to "fix" the memory issue. 
@andrea-pasquale , @Edoardo-Pedicillo, if you could run the autocalibration to update the runcard that would be great

**Comment by andrea-pasquale** [comment]
I've uploaded the recalibrated runcard.
I still need to test the memory issue and the twpa which is currently not addressed by the autocalibration.

**Comment by andrea-pasquale** [comment]
Now the probabilities make more sense.
They are not as good as what @stavros11 was [mentioning](https://github.com/qiboteam/qibolab/actions/runs/4949505803/jobs/8851767053?pr=409), but they are better. Currently we have $p_0 \approx 96 $% and $p_1 \approx 88$%.

**Comment by andrea-pasquale** [comment]
Update currently I get $p_0 \approx 97 $% and $p_1 \approx 93$% with shallow circuits (for p0 I just measure while for p1 I measure after X).

**Comment by rodolfocarobene** [comment]
If possible, I think we should merge this fast to provide a ~good runcard in main

**Comment by andrea-pasquale** [comment]
From my side we can merge.

**Review by andrea-pasquale — APPROVED** [review]

**Pull request merged**